port/raspberrypi/rp2xxx/src/hal/pins.zig

```diff
@@ -509,6 +512,7 @@ pub const GlobalConfiguration = struct {
                     pin.set_pull(.disabled);
                     pin.set_input_enabled(false);
                 } else if (comptime func.is_uart_tx() or func.is_uart_rx()) {
+                    // TODO: Handle pins that can be used as with a alternate uart function
                     pin.set_function(.uart);
```

Comment:
This would fail on rp2350 before since it only had `.uart_first` and `.uart_second`